VERSION

```diff
@@ -1 +1 @@
-15.8.2
+15.8.3
```

Comment:
Can we make this a preview release if we're the only ones interested in it?

Reply:
sure, can do. I do need a release internally but agreed it's not such a big deal for others.